etc/ha_healthchecker/ha_healthchecker.py.j2

```diff
@@ -25,9 +28,15 @@ CONFIG_LOCAL = {
 {% if use_test_account %}
     'github_repo': "{{ test_repo_name }}",
     'github_user_token': "{{ test_github_token }}",
+    'github_user_passwd': "{{ test_github_password }}",
+    'github_user_name': "{{ test_github_user_name }}",
+    'github_app_name': "{{ test_github_app_name }}"
```

Comment:
what's the value of test_github_app_name? we may create one in moo?

Reply:
oh, I forget to add this in the inventory.

Reply:
how about named "test-ha-app"?

Reply:
++